src/components/App.css

```diff
@@ -0,0 +1,252 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+nav,
+ul {
+  display: flex;
```

Comment:
- [ ] As an example here `display: flex;` is used on 11 different places. Kindly create a class for it and call it the react component when needed.